plugins/apitracing/Readme.md

```diff
@@ -1,2 +1,90 @@
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=gdatasoftwareag_apitracing&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=gdatasoftwareag_apitracing)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=gdatasoftwareag_apitracing&metric=coverage)](https://sonarcloud.io/summary/new_code?id=gdatasoftwareag_apitracing)
+
+# ApiTracing Plugin
+
+This plugin hooks functions of specific processes and extracts the function parameters.
+
+## Tracing Config
+
+In the tracing configuration tracing profiles are defined. An example can be
+found [here](configuration/configuration.yml).
```

Comment:
[...]These processes define which functions are to be traced and if additional behavior like tracing children are enabled. [...] Every process being traced, is associated with such a profile. [...]